pkg/assets/kubesaw_admins_test.go

```diff
@@ -0,0 +1,66 @@
+package assets
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCanBeAppliedForDefinedName(t *testing.T) {
+	// given
+	member1Member2 := []string{"member1", "member2"}
+	member2Member3 := []string{"member2", "member3"}
+	testCases := map[string]struct {
+		s               Selector
+		shouldBeSkipped bool
```

Comment:
minor - the `shouldBeSkipped` is tight to the "member1" cluster name which is used at line 35, it wasn't clear at first. Maybe we can move that to the list here and add a parameter like "memberCluster" or somth.

Reply:
I wanted to keep it simple and short - I've rename the test to `TestShouldBeSkippedForMember1` to make it clear